src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/Risks.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
+
+import java.time.LocalDateTime
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Risks as IntegrationApiRisks
+
+data class Risks(
+  val assessedOn: LocalDateTime? = null,
```

Comment:
Can this be of type LocalDateTime straight away or does it need to be a string and then parsed?

Reply:
Yes, it can be of type LocalDateTime straight away.

The webclient runs bodyToMono on the response from the upstream API, which converts it from json into an object (in this case, a Risks object). 

https://github.com/ministryofjustice/hmpps-integration-api/blob/3090c821fc6f0b63f2b1a11da5d1a921cc46fec5/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/extensions/WebClientWrapper.kt#L33-L36

Our gateway tests implicitly test for this, as the mock server stubs the date-time as JSON string.

https://github.com/ministryofjustice/hmpps-integration-api/blob/4b0eec6a479576168c09bdd7fa9e31ec4130678f/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/assessrisksandneeds/GetRisksForPersonTest.kt#L37-L47

Reply:
Good points this makes sense, I wonder if there are unnecessary parses in the codebase where we read time related fields in as Strings when we don't need to. Will make a note of this!